src/spec.md

```diff
@@ -79,6 +79,47 @@ Each `update` which is passed to the callback comes with the following propertie
 - `update.summary`: optional, short text, shown beside icon (see [`sendUpdate()`](#sendupdate))
 
 
+### sendToChat()
+
+```js
+window.webxdc.sendToChat(message);
+```
+
+`sendToChat()` allows an webxdc app to prepare a message
+that can then be sent to a chat by the user.
+Implementations may ask the user for a destination chat
+and then set up the message as a draft,
+so it is clear that the outgoing message is really a result of some user interaction.
+
+- `message`: an object with the following properties:
+  - `message.file.base64`: base64 encoded file data
+  - `message.file.name`: name of the file
+  - `message.text`: message text that should be sent together with the file
+
+Either `message.file`, `message.text` or both needs to be set.
+The text can usually be modified by the user and
+the user may decide to sent the text without the file
+or sent nothing at all.
```

Comment:
"or send nothing at all" ?????

Reply:
yeah :) i added a commit to fix that

Reply:
sorry my ????? was not about the typo (I didn't even realized it) but that I don't understand the meaning of "or send nothing at all"

Reply:
(because if not text nor file is given then calling sendToChat() should fail, not landing in a chat with an empty draft :thinking: )

Reply:
ah now I get it, thanks!